packages/suite/src/components/wallet/WalletLayout/AccountBanners/EvmExplanationBanner.tsx

```diff
@@ -0,0 +1,92 @@
+import styled from 'styled-components';
+import { variables, Icon } from '@trezor/components';
+import { Translation } from 'src/components/suite';
+import { CloseableBanner } from './CloseableBanner';
+import { Account } from 'src/types/wallet';
+import { networks } from '@suite-common/wallet-config';
+import { useSelector } from 'src/hooks/suite/useSelector';
+import { useDispatch } from 'src/hooks/suite/useDispatch';
+import { coinsColors } from '@trezor/theme';
+import { SUITE } from 'src/actions/suite/constants';
+
+const Point = styled.div`
+    display: flex;
+    font-size: ${variables.FONT_SIZE.SMALL};
+    font-weight: ${variables.FONT_WEIGHT.MEDIUM};
+    text-align: center;
+    color: ${({ theme }) => theme.TYPE_LIGHT_GREY};
+
+    & + & {
+        margin-top: 10px;
+    }
+`;
+
+const Dark = styled.span`
+    color: ${({ theme }) => theme.TYPE_DARK_GREY};
+`;
+
+const StyledIcon = styled(Icon)`
+    margin-right: 16px;
```

Comment:
I think it shouldn't be necessary to overload Icon if you will use 
```jsx
gap: ${spacingsPx.md};
```
for `Point`

Reply:
fixed in https://github.com/trezor/trezor-suite/pull/11099/commits/6ee47baa47af021ca155ca51626de835864a58f0